### PR TITLE
ADM-464 feat: upload e2e vidoes artifact

### PR DESCRIPTION
**Pull request by LanxinWang**
## Summary

Screen Shot 2023-05-15 at 15 52 33

### CODE REVIEW

**Review by leeranzhi — APPROVED** [review]

**Review by guzhongren — APPROVED** [review]
Good job

File: .buildkite/pipeline.yml [hunk 1]
```diff
@@ -51,6 +51,7 @@ steps:
 
   - label: ":java: Build Backend"
     branches: main
+    key: "build-backend"
     depends_on: "deploy-infra"
```
Comment (wangwang-zhang) [inline .buildkite/pipeline.yml:55]: Is "depends on" effective?

**Review by wangwang-zhang — APPROVED** [review]
check with comments.

File: .buildkite/pipeline.yml (hunk 1 above)
Comment (leeranzhi) [reply .buildkite/pipeline.yml]: Sure, why not?

**Pull request merged**